internal/errors-migrate/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/upfluence/errors/internal/errors-migrate
+
+go 1.15
```

Comment:
1.16 ?

Reply:
it would work with go1.15+ ( i use go1.15 in local 😛 )